packages/slate-editor/src/extensions/snippet/lib/useFloatingSnippetInput.ts

```diff
@@ -42,8 +42,10 @@ export function useFloatingSnippetInput(editor: Editor): [State, Actions] {
 
         try {
             EditorCommands.insertNodes(editor, node.children);
+            editor.blink(node.children.at(0), node.children.at(-1));
```

Comment:
Have you considered using `Editor.rangeRef` here instead of node objects?

The downside of using nodes, is that if there's any change happens to them after insertion (including normalizations) -- the animation won't work.

Reply:
How rangeRef could help if the nodes are not inserted?